apps/nextjs/src/components/Footer.tsx

```diff
@@ -73,9 +70,6 @@ const Footer = ({ featureFlag }: FooterProps) => {
                           }
                         }}
                         href={tool.href}
-                        disabled={
```

Comment:
What about `!tool.href`? Is href a required field now?

Reply:
The types allowed it to build so I assumed this was fine